v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/options/KafkaToBigQueryFlexOptions.java

```diff
@@ -17,128 +17,134 @@
 
 import com.google.cloud.teleport.metadata.TemplateParameter;
 import com.google.cloud.teleport.v2.kafka.options.KafkaReadOptions;
+import org.apache.beam.runners.dataflow.options.DataflowPipelineOptions;
 import org.apache.beam.sdk.options.Default;
 
 /**
  * The {@link KafkaToBigQueryFlexOptions} class provides the custom execution options passed by the
  * executor at the command-line.
  */
 public interface KafkaToBigQueryFlexOptions
-    extends KafkaReadOptions, BigQueryStorageApiStreamingOptions {
+    extends DataflowPipelineOptions, KafkaReadOptions, BigQueryStorageApiStreamingOptions {
 
-  @TemplateParameter.BigQueryTable(
+  @TemplateParameter.Text(
       order = 1,
+      groupName = "Source",
       optional = true,
-      description = "BigQuery output table",
-      helpText =
-          "BigQuery table location to write the output to. The name should be in the format "
-              + "`<project>:<dataset>.<table_name>`. The table's schema must match input objects.")
-  String getOutputTableSpec();
+      regexes = {"[,:a-zA-Z0-9._-]+"},
+      description = "Kafka Bootstrap Server",
+      helpText = "Kafka Bootstrap Server list, separated by commas.",
+      example = "localhost:9092,127.0.0.1:9093",
+      hiddenUi = true)
```

Comment:
Why is this hidden? looking at template, we don't even need this one right? same goes for `getKafkaReadTopics`

Reply:
I am inheriting `KafkaReadOptions`, so I have to override and hide this, otherwise it would show up in the UI which would be confusing.